src/MeshInertiaCalculator_TEST.cc

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2025 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+#include <gtest/gtest.h>
+
+#include <gz/math/MassMatrix3.hh>
+
+#include "MeshInertiaCalculator.hh"
+
+using namespace gz;
+using namespace sim;
+
+/////////////////////////////////////////////////
+TEST(MeshInertiaCalculator, CorrectMassMatrix)
+{
+  // Verify a mass matrix with a small error can be corrected
+  math::MassMatrix3d massMatrix(55.0,
+                                math::Vector3d(7, 15, 23),
+                                math::Vector3d::Zero);
+  EXPECT_FALSE(massMatrix.IsValid());
+  EXPECT_TRUE(massMatrix.IsPositive());
+  EXPECT_TRUE(MeshInertiaCalculator::CorrectMassMatrix(massMatrix));
```

Comment:
to give coverage for an explicit tolerance value, before this line where you call `CorrectMassMatrix` successfully can you add a call that explicitly passes a tolerance value that is so small that it fails? Then calling it with the default tolerance on this line should still pass

Reply:
yep, updated test. [fb68bd8](https://github.com/gazebosim/gz-sim/pull/2775/commits/fb68bd8d16e97c49088e05dcb34d8faf85b91b62)